src/ts/playerutils.ts

```diff
@@ -106,10 +106,9 @@ export namespace PlayerUtils {
       // Re-evaluate when playback starts
       player.addEventHandler(player.EVENT.ON_PLAY, liveDetector);
 
-      // HLS live detection workaround for Android:
-      // Also re-evaluate during playback, because that is when the live flag might change.
-      // (Doing it only in Android Chrome saves unnecessary overhead on other plattforms)
-      if (BrowserUtils.isAndroid && BrowserUtils.isChrome) {
+      // Live detection workaround for Android and iOS:
+      // It could be that the isLive flag change to late in our native SDK's so we check the status also on time changed
```

Comment:
```suggestion
      // It could be that the isLive flag changes immediately after initialized throughout our native SDKs so we check the status also whenever a time change occurs 
```